src/components/People/CreateAPerson.jsx

```diff
@@ -52,39 +52,35 @@ const CreateAPerson = () => {
     // Required fields must not be null
     personValidationRules.map((rule) => {
       (!(rule.inputField in dataToValidate) || formData[rule.inputField] === '')
-        ? fieldsErroring[rule.inputField] = rule.message
+        ? fieldsErroring[rule.errorDisplayId] = rule.message
         : null;
     });
 
-    // Date fields must be valid
-    if (!(isDateValid(dataToValidate.documentExpiryDateYear, dataToValidate.documentExpiryDateMonth, dataToValidate.documentExpiryDateDay))) {
-      fieldsErroring.documentExpiryDate = 'You must enter a valid date';
+    // DoB must be valid and not in the future
+    if (dataToValidate.dateOfBirthYear || dataToValidate.dateOfBirthMonth || dataToValidate.dateOfBirthDay) {
+      if (!(isDateValid(dataToValidate.dateOfBirthYear, dataToValidate.dateOfBirthMonth, dataToValidate.dateOfBirthDay))
+      || !(isDateBefore(dataToValidate.dateOfBirthYear, dataToValidate.dateOfBirthMonth, dataToValidate.dateOfBirthDay))) {
+        fieldsErroring.dateOfBirth = 'You must enter a valid date of birth';
+      }
     }
```

Comment:
We could make this part a bit more readable, I would also suggest renaming `isDateBefore` to `isInThePast`.
```suggestion
    if (dataToValidate.dateOfBirthYear || dataToValidate.dateOfBirthMonth || dataToValidate.dateOfBirthDay) {
      const isValidFormat = isDateValid(dataToValidate.dateOfBirthYear, dataToValidate.dateOfBirthMonth, dataToValidate.dateOfBirthDay);
      const inThePast = isDateBefore(dataToValidate.dateOfBirthYear, dataToValidate.dateOfBirthMonth, dataToValidate.dateOfBirthDay);
      if (!isValid || !inThePast) {
        fieldsErroring.dateOfBirth = 'You must enter a valid date of birth';
      }
    }
```